sections/sprint-1/networking.tex

```diff
@@ -264,3 +264,62 @@ \subsection{IPv4 and IPv6}\label{ipv4-ipv6}
 IPv6 addresses are usually written as eight 16-bit hexadecimal numbers \cite{socketnetworking}.
 The higher-order bits of the 128-address imply the type.
 For this project, this should not have a significant impact, and either can be used.
+
+
+\subsection{Format of data}\label{subsec:data-format}
+As it is necessary to send data over the network, the format of the data must be decided upon.
+The network format for the field anchor, player position and goal position is \texttt{0xyyyyxxxxiisstt}, where \texttt{t} is the type, \texttt{s} is the timestamp, \texttt{i} is the ID, \texttt{xxxx} is the \texttt{x} position and \texttt{yyyy} is the \texttt{y} position.
+These formats can be seen on \autoref{Tab:anchor-format}, \autoref{Tab:player-format} and \autoref{Tab:player-format}.
+The data sent is almost identical, as the same information is required, the only difference between them is the type and the context of the IDs.
+The type is at the right hand side of the hexadecimal so that it is faster to decode and to switch on which type it is.
+After the type the timestamp is placed, which is used to detect outdated data.
+\\\\
+It was decided to use a size of respectively 16 and 8 bits so that it is straightforward to work with in the \texttt{C\#} language, because it is the same size as some variables in \texttt{C\#} and hereby easy to type cast.
```

Comment:
```suggestion
It was decided to use a size of respectively 16 and 8 bits so that it is straightforward to work with in the \texttt{C\#} language, because it is the same size as some variable types in \texttt{C\#} and hereby easy to type cast.
```